src/cpu/core_normal/table_ea.h

```diff
@@ -136,10 +136,10 @@ static GetEAHandler EATable[512]={
 	EA_16_80_n,EA_16_81_n,EA_16_82_n,EA_16_83_n,EA_16_84_n,EA_16_85_n,EA_16_86_n,EA_16_87_n,
 	EA_16_80_n,EA_16_81_n,EA_16_82_n,EA_16_83_n,EA_16_84_n,EA_16_85_n,EA_16_86_n,EA_16_87_n,
 /* 11 These are illegal so make em 0 */
-	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,
-	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,
-	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,
-	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,	0,
+	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,
+	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,
+	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,
+	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,	nullptr,
 /* 00 */
```

Comment:
Maybe just use single spaces as separators?

Reply:
> Is there any reason I shouldn't just apply clang-format to the whole project except the libs directory in a separate commit?

Yes, don't do that. It can make porting changes over from DOSBox SVN harder than it needs to be.

The "informal" rule of thumb is to reformat only files you're touching, and even then to restrict it to the parts you're touching. And of course ideally always in separate commits.

So we're doing the clang reformatting "lazily", on demand.

Reply:
there is a script that formats only the "changed lines" of the last commit - see: script/format-commit.sh

works for me on Windows only in a MSYS2 environment :/

1. make you're changes
2. commit
3. use the script/format-commit.sh - every changed line gets formatted
(you still need to keep out these tables johnnovak is talking about)
4. rebase or squash the formatted changes back into in one commit

Reply:
That's cool, I never tried it myself. I use a Vim plugin for Clang Format and pretty much reformat what I've changed manually 100% of the time.

Reply:
i was forced to use it - for my micro pulls :) - nothing compared to your output